pcsx2/CDVD/CsoFileReader.cpp

```diff
@@ -39,12 +40,12 @@ struct CsoHeader
 
 static const u32 CSO_READ_BUFFER_SIZE = 256 * 1024;
 
-bool CsoFileReader::CanHandle(const wxString& fileName)
+bool CsoFileReader::CanHandle(const std::string& fileName, const std::string& displayName)
 {
 	bool supported = false;
-	if (wxFileName::FileExists(fileName) && fileName.Lower().EndsWith(L".cso"))
+	if (StringUtil::EndsWith(displayName, ".cso"))
```

Comment:
Is this intended? Shouldn't this check `fileName`? Or maybe both are fine?

Reply:
The aim was to support protocols/file systems where the display name is different from the file name, so the extension is checked based on the "visible" name, but the "internal" name is used to open it.